### DISCUSSION TITLE
Is there a possibility of osu implementing guide sound?

### DISCUSSION THREAD

**Discussion by Alsen99** [ppy/osu#24533] [Ideas]
Guide sound or Answer sound is like hitsound but separate from player's input, it also has a different sound from hitsounds.The first rhythm game I see this being implemented is in maimai. The purpose of this is to guide the player through the rhythm and also help to learn the rhythm. This will help in taiko and osu standard since my hitsound have a noticeable delay and throws me off.

**Comment by Rekunan** [comment]
Can you give a video example of this? I'm having trouble understanding what this is.

**Reply by ItsShamed** [reply]
I think they want another set of hitsounds like in ADOFAI where the “hitsounds” are not based on player input but fully based on the objects' timing and have the same base offset as the track.

For example, here's a chart of “Artificial Snow” by Camellia in ADOFAI:

> [!NOTE]
> As you can see, regardless of the timing of my hits, the hitsounds still play in time.

https://github.com/ppy/osu/assets/39100084/dac14fa3-b113-4bd4-84ba-62e7b5eb8b12

Here is the original track:

> [!NOTE]
> Notice how the original track lacks of the syncopated rhythm the ADOFAI chart has, this is because the hitsounds kind of “overplay” the track.

https://github.com/ppy/osu/assets/39100084/1e6976d7-b98a-49ff-b1c0-6ac5830bcde7

**Reply by Alsen99** [reply]
Yes!

**Reply by Rekunan** [reply]
Ahh, I understand now, thank you.

With the way people currently use hitsounds, I think this would be counterintuitive. I see a lot of people raise their effect volume compared to music to hear their hitsounds better, helping with timing and rhythm. If instead they were guide sounds, you get no audible feedback on how early or late you tap.

> my hitsound have a noticeable delay

Isn't this an issue with your audio latency/offset?

**Reply by TwoQuantumBits** [reply]
I think that guide sounds have some use - otherwise arcade rhythm games wouldn't even have them in the first place:

> I see a lot of people raise their effect volume compared to music to hear their hitsounds better, helping with timing and rhythm.

In my opinion, this is mostly true provided that your timing is consistent most of the time. If your timing is not consistent, however, the hitsounds might actually throw you off instead (especially on hitsounded maps).

> If instead they were guide sounds, you get no audible feedback on how early or late you tap.

If audio is calibrated correctly, you can still get feedback by comparing the timing of the hitsound to the guide sound.



With all of that said, I still feel like there isn't much demand for guide sounds from within the community at the moment. The truth might be different, but I haven't looked into that yet.

**Comment by monochrome22** [comment]
Duplicate of https://github.com/ppy/osu/discussions/19915 if I understand correctly.

**Reply by Rekunan** [reply]
Yep seems like it.
